polaris/polaris/integrations/transactions.py

```diff
@@ -19,8 +26,8 @@ def poll_pending_deposits(cls, pending_deposits: QuerySet
         **OVERRIDE REQUIRED**
 
         This function should poll the financial entity for the state of all
-        `pending_deposits` transactions and return the ones ready to be
-        executed on the Stellar network.
+        `pending_deposits` and return the ones ready to be executed on the
```

Comment:
Clarify that it returns the ones where the external deposit has been completed.  "ready to be executed on the stellar network" implies that the external deposit has been completed but it couldn't help to be explicit